packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -761,6 +773,13 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     }
   }
 
+  /** @private */
+  _itemClassNameGeneratorChanged(generator, oldGenerator) {
```

Comment:
nit: could align naming with the same function in combo-box and use `__itemClassNameGeneratorChanged`

Reply:
Done.